pom.xml

```diff
@@ -1818,6 +1818,27 @@
     </build>
 
     <profiles>
+        <profile>
+            <id>jdk17</id>
+            <activation>
+                <jdk>[17,)</jdk>
+            </activation>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-surefire-plugin</artifactId>
+                        <configuration>
+                            <!-- This is required for Phoenix tests to work -->
```

Comment:
This is required by the JVM in which Phoenix runs - correct? Trino isn't the one performing these accesses?

Reply:
Yes. Exactly

Reply:
Thanks, we'll need to mention this in release notes then when we switch to JDK 17 (by merging this PR)